apps/fluent-tester/src/testPages.ts

```diff
@@ -1,183 +1,249 @@
-import { MenuTest, HOMEPAGE_MENU_BUTTON } from './TestComponents/Menu';
 import { NotificationTest, HOMEPAGE_NOTIFICATION_BUTTON } from './TestComponents/Notification';
+import { PersonaTest, HOMEPAGE_PERSONA_BUTTON } from './TestComponents/Persona';
+import { PersonaCoinTest, HOMEPAGE_PERSONACOIN_BUTTON } from './TestComponents/PersonaCoin';
+import { PressableTest, HOMEPAGE_PRESSABLE_BUTTON } from './TestComponents/Pressable';
+import { RadioGroupTest, HOMEPAGE_RADIOGROUP_BUTTON } from './TestComponents/RadioGroup';
+import { SeparatorTest, HOMEPAGE_SEPARATOR_BUTTON } from './TestComponents/Separator';
+import { ShadowTest, HOMEPAGE_SHADOW_BUTTON } from './TestComponents/Shadow';
+import { ShimmerTest, HOMEPAGE_SHIMMER_BUTTON } from './TestComponents/Shimmer';
+import { SvgTest, HOMEPAGE_SVG_BUTTON } from './TestComponents/Svg';
+import { SwitchTest, HOMEPAGE_SWITCH_BUTTON } from './TestComponents/Switch';
+import { TabsTest, HOMEPAGE_TABS_BUTTON } from './TestComponents/Tabs';
+import { ExperimentalTabsTest, HOMEPAGE_EXPERIMENTAL_TABS_BUTTON } from './TestComponents/TabsExperimental';
+import { TextTest, HOMEPAGE_TEXT_BUTTON } from './TestComponents/Text';
+import { TextExperimentalTest, HOMEPAGE_EXPERIMENTAL_TEXT_BUTTON } from './TestComponents/TextExperimental';
+import { ThemeTest, HOMEPAGE_THEME_BUTTON } from './TestComponents/Theme';
+import { TokenTest, HOMEPAGE_TOKEN_BUTTON } from './TestComponents/Tokens';
 
 export const tests: TestDescription[] = [
   {
-    name: 'ActivityIndicator Test',
+    name: 'ActivityIndicator',
     component: ActivityIndicatorTest,
     testPage: HOMEPAGE_ACTIVITY_INDICATOR_BUTTON,
+    platforms: ['android', 'ios', 'macos', 'web', 'win32', 'windows'],
+  },
+  {
+    name: 'Avatar',
+    component: AvatarTest,
+    testPage: HOMEPAGE_AVATAR_BUTTON,
+    platforms: ['android', 'ios', 'macos', 'web', 'win32', 'windows'],
   },
   {
-    name: 'Avatar Test',
+    name: 'NativeAvatar',
     component: NativeAvatarTest,
     testPage: HOMEPAGE_NATIVE_AVATAR_BUTTON,
+    platforms: ['ios', 'macos'],
   },
   {
-    name: 'Badge Test',
+    name: 'Badge',
     component: BadgeTest,
     testPage: HOMEPAGE_BADGE_BUTTON,
+    platforms: ['android', 'ios', 'macos', 'web', 'win32', 'windows'],
   },
   {
-    name: 'Button Test',
+    name: 'Button',
     component: ButtonTest,
     testPage: HOMEPAGE_BUTTON_BUTTON,
+    platforms: ['android', 'ios', 'macos', 'web', 'win32', 'windows'],
   },
   {
-    name: 'Focus Trap Zone Test',
+    name: 'Callout',
+    component: CalloutTest,
+    testPage: HOMEPAGE_CALLOUT_BUTTON,
+    platforms: ['android', 'macos', 'win32', 'windows'],
+  },
+  {
+    name: 'Checkbox',
+    component: CheckboxTest,
+    testPage: HOMEPAGE_CHECKBOX_BUTTON,
+    platforms: ['android', 'ios', 'macos', 'web', 'win32', 'windows'],
+  },
+  {
+    name: 'Experimental Checkbox',
```

Comment:
Could we rename all the experimental stuff to "Checkbox (Experimental)" so that when we sort components alphabetically, they show up next to each other? :)